actions/commit-and-push/action.yaml

```diff
@@ -0,0 +1,37 @@
+name: "Commit and push"
+description: "Commits all the changed files and pushes them to the repository"
+
+inputs:
+  ref:
+    description: "The ref name to commit and push the files on."
+    required: true
```

Comment:
Can you please use the current branch here as default and set `required: false`?

Reply:
Sounds like a good idea but it seems that the `$default-branch` variable can be used in Workflow Templates but not sure if they work in the composite action.
here is the link:
https://stackoverflow.com/a/65723433

Reply:
Try if `$default-branch` works, otherwise you can get the default branch like this if the input is empty:
```${{ github.event.repository.default_branch }}```

Reply:
@yannick-roeder The `${{ github.event.repository.default_branch }}` can be passed as the default input value in the input of the composite action! :tada: